flink/v1.12/flink/src/main/java/com/netease/arctic/flink/read/hybrid/enumerator/ArcticSourceEnumState.java

```diff
@@ -33,14 +34,18 @@ public class ArcticSourceEnumState {
   private final Collection<ArcticSplitState> pendingSplits;
   @Nullable
   private final long[] shuffleSplitRelation;
+  @Nullable
+  private final FirstSplits firstSplits;
```

Comment:
`FirstSplits` only if for temporal join ? if yes, named `TemporalJoinSplit` is better IMO.